SS_popdyn.tpl

```diff
@@ -156,14 +153,16 @@ FUNCTION void get_initial_conditions()
   if (Hermaphro_Option != 0)
     get_Hermaphro();
 
-  if (MG_active(1) > 0)
+  if (do_once>0 || MG_active(1) > 0)
```

Comment:
checking that || is right and shouldn't be &&. 
Also, style: missing spaces. `do_once > 0`